Nos/Service/Relay/Filter.swift

```diff
@@ -14,7 +14,10 @@ struct Filter: Hashable, Identifiable {
     var limit: Int?
     var since: Date?
     var until: Date?
-    
+
+    /// This filter should keep the subscription open.
+    var subscribe: Bool
+
     init(
```

Comment:
This has a lot of parameters, but I generally like to have doc comments for all of them on non-private `init`s (and really anything that's non-private). One of the things I'd really like to have here in the docs would be the sentence "Defaults to `true`." for the `subscribe` parameter. "Defaults to ..." comments for the others would be great, too.

Reply:
That's a good idea. I didn't add any of them but I think I know what each one does in order to write meaningful docs for all of them.